coverage.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Script which generates coverage information given a configuration *.yml file.
+# 
+# Usage: coverage.sh default.yml
+# Output: coverage_default directory which contains coverage in html format
+#
+# Author: Oliver Fuhrer, Vulcan Inc.
+
+# stop on errors
+set -e
+
+# arguments
+config_file=$1
+if [ -z "${config_file}" -o ! -f "${config_file}" ] ; then
+  echo "Error: must specify a valid config file as argument (usage: $0 config.yml)"
+  exit 1
+fi
+
+# setup
+base_dir=`pwd`
+config_name=`basename ${config_file} .yml`
```

Comment:
why is there a space before the .yml?

Reply:
Command basename removed the extension ".yml" from the ${config_file}.